src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age
+      ? <p className="Person__age">{`I am ${person.age}`}</p>
+      : null
+    }
+    <p className="Person__partner">
+      {person.isMarried
+        ? `${person.partnerName}  is my ${person.sex === 'm'
+          ? `wife`
+          : ' husband'}`
+        : 'I am not married'
+    }
```

Comment:
Could you improve this part a little bit?
It's a bad practice to use nested ternary operators. It's better to move this logic to the helper. 
For example:
```
const getMarriedText = (person) => {
  if (!person.isMarried) {
    return 'I am not married';
  }

  return // use your ternary solution
```